code/modules/mob/living/brain/life.dm

```diff
@@ -4,6 +4,17 @@
 		return
 	if(!loc)
 		return
+	// this is a sanity check to make sure that our MMI reference is still valid
+	// there used to be a bug where container still held a reference to an MMI that the brain wasn't
+	// inside of which caused the brain mob to be permanently stuck in nullspace even with a client,
+	// so this code exists as a failsafe in case everything goes wrong
```

Comment:
you can safely discard these three lines, we're only concerned about the present, not the past. if they really cared they would look at the associated git commit message or your PR